_articles/metrics.md

```diff
@@ -73,6 +74,10 @@ For example, if your project lands on the front page of Hacker News, you'll prob
 
 Try to figure out where your audience is coming from and ask others for feedback on your project page to figure out which of these two issues you're facing.
 
+Once you know that people are using your project, you might want to try to figure out what they are doing with it.  Are they building on it by forking your code and adding features?  Are they using it for science or business?
+
+In order to determine this, you can use tools like [depsy](http://depsy.org), or you can [publish your software and make it citable](https://guides.github.com/activities/citable-code/).  Then if you also ask users to cite your code, particularly in academia, you can use existing citation tools (such as [Google Scholar](https://scholar.google.com)) to understand what users say they are doing with it.
```

Comment:
I agree with your ideally, but will remove the paragraph, since you are in charge, and I don't have any good more general examples.